roles/provision-data-sync-apb/tasks/provision-sync-server.yml

```diff
@@ -28,6 +28,20 @@
     tls_insecure_edge_termination_policy: Redirect
   register: data_sync_server_route
 
+- name: "Create ImageStream with tag {{ sync_server_version }} for Data Sync Server image"
+  openshift_v1_image_stream:
+    name: "data-sync-server"
+    namespace: "{{ namespace }}"
+    labels:
+      app: "data-sync"
+    lookup_policy_local: False
```

Comment:
I see that most of our APB's have this setup. Really nice feature.
https://github.com/ansible/ansible-kubernetes-modules/blob/master/library/openshift_v1_image_stream_tag.py#L160